src/modules/45-dashboards/components/DashboardResourceModalBody/DashboardResourceModalBody.tsx

```diff
@@ -84,18 +89,15 @@ const DashboardResourceModalBody: React.FC<RbacResourceModalProps> = ({
   const [page, setPage] = React.useState(0)
   const { getString } = useStrings()
 
-  const { data: folders, loading: fethingFolders } = useGet({
-    // Inferred from RestfulProvider in index.js
-    path: 'gateway/dashboard/folder',
+  const { data: folders, loading: fethingFolders } = useGetFolder({
```

Comment:
typo in "fethingFolders"

```suggestion
  const { data: folders, loading: fetchingFolders } = useGetFolder({
```